helm/microcosm/templates/cgimap-deployment.yaml

```diff
@@ -32,8 +32,8 @@ spec:
           command:
             - "/app/start.sh"
           args:
-            - "--port=8000"
-            - "--instance=30"
+            - --port={{ .Values.ingestion.pods.cgimap.args.PORT }}
```

Comment:
restore the port back to its original value, there is not point changing at, and even if you do, you also need to change it in nginx.

Reply:
are you serious? 
![image](https://user-images.githubusercontent.com/48960890/114039621-930e5c00-988b-11eb-8f39-89843ebbf991.png)

Reply:
pretty sure i marked only line 36, if not my bad.

Reply:
also this is why I wrote value and not values.

Reply:
oh

Reply:
done